boards/shields/rtkmipilcdb00000be/doc/index.rst

```diff
@@ -0,0 +1,61 @@
+.. _rtkmipilcdb00000be:
+
+RTKMIPILCDB00000BE MIPI Display
+############################
+
+Overview
+********
+
+The Focus LCDs RTKMIPILCDB00000BE MIPI Display is a 4.5 inch TFT 480x854 pixels
+capacitive touch panel, and a backlight unit.
+
+This display uses a 26 pin connector header.
+
+Pins Assignment of the Renesas RTKMIPILCDB00000BE MIPI Display
+===============================================================
```

Comment:
Underlines need to be the same width as the title (it looks like you have an extra `=` here, at least in the github UI)